src/modules/commander/commander.cpp

```diff
@@ -185,6 +185,9 @@ static bool _last_condition_global_position_valid = false;
 
 static struct vehicle_land_detected_s land_detector = {};
 
+static float _eph_threshold_adj = INFINITY;	///< maximum allowable horizontal position uncertainty after adjustment for flight condition
```

Comment:
Can these be Commander member variables instead of additional statics? I also think we might not even need the _skip_pos_accuracy_check (mentioned below).